src/mmw/js/src/analyze/views.js

```diff
@@ -300,8 +300,23 @@ var TabPanelView = Marionette.ItemView.extend({
         role: 'presentation'
     },
 
+    ui: {
+        'a': 'a',
```

Comment:
Will `[data-toggle=tab]` select the same element? Might be a little more descriptive. Or we could name the `ui` key something other than `a`.

Reply:
So rename either one? You don't like the same key being used in both cases, or would like both to be more descriptive?

Reply:
I wasn't sure what element the `'a'` referred to without looking at the template, so thought it might help to either change what we're selecting on, or rename what we've selected, or both. Also, happy to have it left as is.